source/_integrations/izone.markdown

```diff
@@ -4,7 +4,7 @@ description: Instructions on how to integrate iZone climate control devices with
 ha_category:
   - Climate
 ha_release: '0.100'
-ha_iot_class: Local Push
+ha_iot_class: Local Polling
```

Comment:
If I remember correctly, these are synced over from the core so should be updated in `izone/manifest.json`

Reply:
Ah, I did that already. There's a PR with this exact change pending. It is against the current branch because that's what's currently happening.

Reply:
[Here's the PR](https://github.com/home-assistant/core/pull/61548).

Reply:
It is fine to have a PR though, makes the `next` branch viewed correctly :)